Week3/homework/exercise-3.js

```diff
@@ -0,0 +1,34 @@
+'use string';
+// Exercise 3: Be your own fortune teller
+
+// Why pay a fortune teller when you can just program your fortune yourself?
+
+// Write a function named tellFortune.
+// It takes 4 arguments: number of children (number), partner's name (string), geographic location (string), job title (string).
+// Randomly select values from the arrays.
+// Return a string: "You will be a [JOB_TITLE] in [LOCATION], and married to [PARTNER_NAME] with [NUMBER_KIDS] kids."
+// Create 4 arrays, numChildren, partnerNames, locations and jobs. Give each array 5 random values that make sense
+// Call the function 1 time, by passing the arrays as the argument.
+
+//i need a function that selects random number so i've declared it.
+function randomElement(array) {
+  let randomIndex = Math.floor(Math.random() * array.length);
+  return array[randomIndex];
+}
```

Comment:
Nice work splitting this into a separate function! You can really see how using functions allows you to write re-usable code.

Reply:
Yes i 've learned it. Thank you